examples/backbone_marionette/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "backbone.marionette.todo.mvc",
```

Comment:
`name` is optional when not publishing. Make sure you're on the latest npm version.

This also needs a `"private": true,`.